tce/src/base/program/BasicBlock.hh

```diff
@@ -108,6 +108,16 @@ public:
     unsigned tripCount() const { return tripCount_; }
     void setTripCount(unsigned count) { tripCount_ = count; }
 
+    /// BB is known to be a hwloop body
```

Comment:
Why duplicate this in BasicBlockNode? Why not call the BB method? Is it needed outside scheduling (thus does BasicBlock need it really or BasicBlockNode would be enough)?

Reply:
You are right! We can get away with just having this on BasicBlockNode. I have updated the patch for it.